#! /bin/sh /usr/share/dpatch/dpatch-run
## 80_fix_tac_json.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix for https://dev.icinga.org/issues/2160

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' pkg-icinga~/cgi/tac.c pkg-icinga/cgi/tac.c
--- pkg-icinga~/cgi/tac.c	2011-11-26 09:13:04.000000000 +0100
+++ pkg-icinga/cgi/tac.c	2012-03-31 10:44:49.921561000 +0200
@@ -1710,8 +1710,8 @@
 		printf("\"total_active_host_checks_with_passive_disabled\": %d,\n", total_active_host_checks_with_passive_disabled);
 
 		printf("\"total_active_service_checks\": %d,\n", total_active_service_checks);
-		printf("\"total_passive_service_checks\": %d\n", total_passive_service_checks);
-		printf("\"total_disabled_service_checks\": %d\n", total_disabled_service_checks);
+		printf("\"total_passive_service_checks\": %d,\n", total_passive_service_checks);
+		printf("\"total_disabled_service_checks\": %d,\n", total_disabled_service_checks);
 		printf("\"total_active_service_checks_with_passive_disabled\": %d\n", total_active_service_checks_with_passive_disabled);
 
 		printf(" }\n");
